### PR TITLE
completed sending requests to api

### PR DESCRIPTION
**Pull request by mrSkyGuy**
Жду 20000 рублей на карту до 11 вечера этого дня

### CODE REVIEW

File: templates/index.html [hunk 1]
```diff
@@ -77,17 +77,55 @@
...
+
+            xhr.onerror = () => reject(xhr.response)
+
+            method.toLowerCase() == "post" ? 
+                xhr.send(JSON.stringify(body)) 
+                : xhr.send()
+        })
+    } 
+
+    const request = sendRequest("GET", "127.0.0.1:8080/api/presets")
```
Comment (platon-p) [inline templates/index.html:121]: HTTP protocol missed

File: templates/index.html [hunk 2]
```diff
@@ -77,17 +77,55 @@
...
+    const fioInput = document.querySelector("#fio_input")
+    const maleButton = document.querySelector("#male")
+    const femaleButton = document.querySelector("#female")
+    const selectList = document.querySelector("#select")
+    const submitButton = document.querySelector("#submit")
 
-    submit_button.onclick = function () {
-        path = "?name=" + fio_input.value.split(' ').join('+') + "&gender=" + (male_button ? "male" : "female") + "&type=" + select_list.value
+    submitButton.addEventListener('click', () => {
+        path = "?name=" + fio_input.value.split(' ').join('+')
```
Comment (platon-p) [inline templates/index.html:87]: the const name has been changed, but the name in the function has not changed

File: templates/index.html (hunk 1 above)
Comment (mrSkyGuy) [reply templates/index.html:121]: Sorry daddy

File: templates/index.html (hunk 2 above)
Comment (mrSkyGuy) [reply templates/index.html:87]: For what?

**Pull request closed without merging**